packages/manager/apps/dedicated/client/app/dedicated/server/mrtg/mrtg.controller.js

```diff
@@ -0,0 +1,199 @@
+            scaleLabel: {
+              display: true,
+              labelString: yLabel,
+            },
+          },
+        ],
+        xAxes: [
+          {
+            scaleLabel: {
+              display: true,
+              labelString: this.$translate.instant(
+                'server_usage_chart_xaxis_label',
+              ),
+            },
+          },
+        ],
+      },
+    };
+  }
+
+  loadStatistics() {
+    const nameServer = null;
+    this.serverStatsLoad.loading = true;
+    this.serverStatsLoad.error = false;
+    const promises = {
+      interfaces: this.Server.getNetworkInterfaces(this.$stateParams.productId),
+      statConst: this.Server.getStatisticsConst(),
+    };
+
+    this.$q
+      .all(promises)
+      .then((stats) => {
+        const hasVrack = some(
+          stats.interfaces,
+          (networkInterface) => networkInterface.linkType === 'private',
+        );
+        if (!hasVrack) {
+          stats.interfaces.push({
+            linkType: 'no_vrack',
+            disabled: true,
+          });
+        }
+
+        this.networks = map(stats.interfaces, (networkInterface) => ({
+          id: networkInterface.mac,
+          linkType: networkInterface.linkType,
+          displayName: this.$translate.instant(
+            `server_tab_stats_network_${networkInterface.linkType}`,
+            {
+              t0: networkInterface.mac,
+            },
+          ),
+          disabled: networkInterface.disabled,
+        }));
+
+        this.serverStats.networkChoice = this.networks[0].id;
+
+        this.serverStats.modelconst = stats.statConst;
+
+        if (this.serverStats.periodeChoice === null) {
+          this.serverStats.periodeChoice = stats.statConst.defaultPeriod;
+        }
+
+        if (this.serverStats.typeChoice === null) {
+          this.serverStats.typeChoice = stats.statConst.defaultType;
+        }
+
+        return this.Server.getStatistics(
+          this.$stateParams.productId,
+          this.serverStats.networkChoice,
+          this.serverStats.typeChoice,
+          this.serverStats.periodeChoice,
+        );
+      })
+      .then((statistics) => {
+        this.serverStats.model = statistics;
+        this.createChart(statistics);
+      })
+      .catch((data) => {
+        this.serverStatsLoad.error = true;
+
+        if (nameServer && data) {
+          this.dedicatedServer.setMessage(
+            this.$translate.instant('server_tab_STATS_loading_fail'),
+            {
+              ...data.data,
+              typpe: 'ERROR',
+            },
+          );
+        }
+      })
+      .finally(() => {
+        this.serverStatsLoad.loading = false;
+      });
+  }
+
+  getStatistics() {
+    this.serverStatsLoad.loading = true;
+    this.serverStatsLoad.error = false;
+
+    this.Server.getStatistics(
+      this.$stateParams.productId,
+      this.serverStats.networkChoice,
+      this.serverStats.typeChoice,
+      this.serverStats.periodeChoice,
+    )
```

Comment:
This code appears several times so create a function instead to avoid duplication
For example : 
```suggestion
    this.getServerStatistics()
```

Reply:
I am not sure whether it's correct or not because there is already a getStatistics function with the same name that doesn't take any argument (your suggestion is currently in that function). Would it make an infinite loop ?
Anyway, this code is not new, it's just a move from the dashboard/dashboard.controller.js....

Reply:
@JayBeeDe can you please take a look on @darsene's comment?

Reply:
done, sorry for delay